awx/main/dispatch/worker/callback.py

```diff
@@ -52,7 +53,7 @@ def __init__(self):
 
     def read(self, queue):
         try:
-            res = self.redis.blpop(settings.CALLBACK_QUEUE, timeout=settings.JOB_EVENT_BUFFER_SECONDS)
+            res = self.redis.blpop(settings.CALLBACK_QUEUE, timeout=1)
```

Comment:
@chrismeyersfsu I think we should just set this to one, as `blpop` doesn't let you set this lower.  This is mostly to avoid a scenario where we have a buffer, and the read from redis hangs.